config/reference_config.go

```diff
@@ -171,6 +171,9 @@ func (refconfig *ReferenceConfig) getUrlMap() url.Values {
 
 	//filter
 	urlMap.Set(constant.REFERENCE_FILTER_KEY, mergeValue(consumerConfig.Filter, refconfig.Filter, constant.DEFAULT_REFERENCE_FILTERS))
+	if refconfig.Generic {
+		urlMap.Set(constant.REFERENCE_FILTER_KEY, mergeValue(consumerConfig.Filter, refconfig.Filter, constant.GENERIC_REFERENCE_FILTERS))
+	}
```

Comment:
I think the `refconfig.Generic` should be set to url. And check it when genericfilter.

Reply:
i add generic to url; when generic is true,add genericfilter to filter